pkg/gofr/gofr.go

```diff
@@ -92,9 +95,53 @@ func New() *App {
 
 	app.subscriptionManager = newSubscriptionManager(app.container)
 
+	// static fileserver
+	currentWd, _ := os.Getwd()
+	checkDirectory := filepath.Join(currentWd, publicDir)
+
+	if _, err = os.Stat(checkDirectory); err == nil {
+		app.AddStaticFiles(publicDir, checkDirectory)
+	}
+
 	return app
 }
 
+func updateConfigInformation(config *gofrHTTP.StaticFileConfig, envConfig config.Config) {
+	config.DirectoryListing, _ = strconv.ParseBool(envConfig.GetOrDefault("STATIC_DIRECTORY_LISTING", "true"))
+	config.HideDotFiles, _ = strconv.ParseBool(envConfig.GetOrDefault("STATIC_HIDEDOTFILES", "true"))
+	config.ExcludeExtensions = strings.Split(envConfig.Get("STATIC_EXCLUDE_EXTENSIONS"), ",")
+	config.ExcludeFiles = strings.Split(envConfig.Get("STATIC_EXCLUDE_FILES"), ",")
+	config.ExcludeFiles = append(config.ExcludeFiles, "openapi.json")
+}
+
+func (a *App) AddStaticFiles(endpoint, filePath string) {
+	a.httpRegistered = true
+
+	var dupFilePath string
+
+	defaultConfig := a.httpServer.router.GetDefaultStaticFilesConfig()
+
+	updateConfigInformation(&defaultConfig, a.Config)
+
+	if strings.HasPrefix(filePath, "./") {
+		dupFilePath, _ = os.Getwd()
+		dupFilePath = filepath.Join(dupFilePath, filePath)
+	} else {
+		dupFilePath = filePath
```

Comment:
I would remove the else block and define dupfilePath := filepath before the if that check fie relative path

Reply:
made dupFilePath as mentioned